src/code/reducers/helpers/load-state-from-authoring.js

```diff
@@ -113,34 +140,9 @@ export function loadNextTrial(state, authoring, progress) {
   if (state.trialSuccess){
     trial = (state.trial < state.trials.length) ? (trial+1) : 1;
```

Comment:
You didn't change this line, so this comment is not technically related to this PR, but if the point of this line is to wrap around, shouldn't it wrap to trial 0?